h2/src/main/org/h2/mvstore/rtree/SpatialKey.java

```diff
@@ -121,4 +132,34 @@ public boolean equalsIgnoringId(SpatialKey o) {
         return Arrays.equals(minMax, o.minMax);
     }
 
+
+    @Override
+    public StringBuilder getSQL(StringBuilder builder) {
+        return null;
+    }
+
+    @Override
+    public TypeInfo getType() {
+        return null;
+    }
+
+    @Override
+    public int getValueType() {
+        return 0;
+    }
+
+    @Override
+    public String getString() {
+        return null;
+    }
+
+    @Override
+    public Object getObject() {
+        return null;
+    }
+
+    @Override
+    public void set(PreparedStatement prep, int parameterIndex) throws SQLException {
+
+    }
```

Comment:
`Value` is a new parent of this class and implementation in `Value` throws an exception. Perhaps this no-op implementation should be removed from `SpatialKey`, because it can hide unexpected illegal invocation of this method.